docs/source/specs/openapi.json

```diff
@@ -82,6 +82,15 @@
               "type": "string"
             }
           },
+          {
+            "in": "query",
+            "name": "org_id",
```

Comment:
I don't think this behavior actually exists in this PR? The above `account` filter is enabled by this: https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/api/cross_access/view.py#L50-L53 to allow for `/cross-account-requests/?account=123`

Reply:
Ok you mentioned in a comment above that I needed to add org_id to the spec at https://github.com/RedHatInsights/insights-rbac/blob/master/docs/source/specs/openapi.json#L48 .

Reply:
Sorry, yeah I was just linking to the resource, but wasn't explicit enough clarifying which actions we needed it on.